JitsiConference.js

```diff
@@ -47,6 +46,11 @@ function JitsiConference(options) {
     this._init(options);
     this.componentsVersions = new ComponentsVersions(this);
     this.participants = {};
+    /**
+     * Jingle Session instance
+     * @type {JingleSessionPC}
+     */
+    this.jingleSession = null;
```

Comment:
Yes, previously JingleSession was stored in ChatRoom. I moved it in this commit:
https://github.com/jitsi/lib-jitsi-meet/pull/362/commits/bee0b28b933807272bd693e5f35429feb25f6688